src/main/java/com/amazon/aocagent/testbeds/EKSTestBed.java

```diff
@@ -0,0 +1,21 @@
+package com.amazon.aocagent.testbeds;
+
+import com.amazon.aocagent.helpers.EKSTestOptionsValidationHelper;
+import com.amazon.aocagent.models.Context;
+import lombok.extern.log4j.Log4j2;
+
+@Log4j2
+public class EKSTestBed implements TestBed {
+  private Context context;
+
+  @Override
+  public void init(Context context) {
+    EKSTestOptionsValidationHelper.checkEKSTestOptions(context);
+    this.context = context;
+  }
+
+  @Override
+  public Context launchTestBed() throws Exception {
+    return this.context;
```

Comment:
can we add a todo comment here to indicate we will make eks cluster creation in setup?

Reply:
The current plan is to ask user to use their own cluster, they can simply use eksctl to create one.